AlphaWallet/Transfer/ViewControllers/SendViewController.swift

```diff
@@ -47,6 +47,7 @@ class SendViewController: UIViewController, CanScanQRCode {
     private let recipientViewModel = SendViewSectionHeaderViewModel(
         text: R.string.localizable.sendRecipient().uppercased()
     )
+    private weak var invalidTokenAlert: UIViewController?
```

Comment:
Can you add a comment above this line to explain that it is relying on the `weak var` being nil-ed when the alert closes, to make sure that at most 1 instance of the alert is shown at any time?

Reply:
Done